onevm/gratia/onevm/process_utils.py

```diff
@@ -34,51 +25,12 @@ def iexe_cmd(cmd, stdin_data=None):
     error_lines = None
     exit_status = 0
     try:
-        # XXX: no popen2 in python3
-        child = popen2.Popen3(cmd, capturestderr=True)
+        p = Popen(cmd, shell=True, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+        out, err = p.communicate(stdin_data)
```

Comment:
Careful: communicate() takes and returns bytes here.

Reply:
Right...

Reply:
I am going to wimp out and use universal_newlines here